src/base/utils/fs.cpp

```diff
@@ -287,8 +287,12 @@ QString Utils::Fs::branchPath(const QString &filePath, QString *removed)
     return ret;
 }
 
-bool Utils::Fs::sameFileNames(const QString &first, const QString &second)
+bool Utils::Fs::sameFileNames(QString first, QString second)
 {
+    if (first.endsWith(QB_EXT, Qt::CaseInsensitive))
```

Comment:
I don't think this function should handle this. If you really need this, you should create another function.

Reply:
I succumbed to the corrupting influence of the fact that now `.!qb` extension handling can be found anywhere...